deployment.rb

```diff
@@ -187,6 +187,18 @@ def hourofcode_hostname
     canonical_hostname('hourofcode.com')
   end
 
+  def circle_run_identifier
+    ENV['CIRCLE_BUILD_NUM'] ? "CIRCLE-BUILD-#{ENV['CIRCLE_BUILD_NUM']}-#{ENV['CIRCLE_NODE_INDEX']}" : nil
```

Comment:
@islemaster I think CircleCI doesn't hit that code path (that's the one used by the main staging/test/prod machines), and we could always (/do?) rely on Circle's build artifact system to retain those logs.